### PR TITLE
[FEATURE] MS-30: Companies with Redux

### PR DESCRIPTION
**Pull request by Sabrucita**

### CODE REVIEW

File: src/redux/companies/actions.js [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+export const getCompaniesFulfilled = (payload) => ({
+  type: GET_COMPANIES_FULFILLED,
+  payload
+});
+
+export const getCompaniesRejected = (payload) => ({
+  type: GET_COMPANIES_REJECTED,
+  payload
+});
```
Comment (BaltazarCortina) [inline src/redux/companies/actions.js:35]: Payload is not being used in the reducer, it's not necessary to pass it as a param. Same for all the 'rejected' actions

File: src/redux/companies/thunks.js [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+            contactFullName: data.contactFullName,
+            contactPhone: parseInt(data.contactPhone),
+            isActive: data.isActive
+          };
+          return dispatch(getCompanyFulfilled(currentData));
+        }
+      })
+      .catch((err) => {
+        const error = { error: true, msg: err };
+        dispatch(getCompanyRejected(error));
```
Comment (BaltazarCortina) [inline src/redux/companies/thunks.js:50]: Line 49 is not necessary, and we don't need error as a param. Same for all the 'rejected' actions.
We could show the error in the modal instead.

File: src/redux/companies/actions.js (hunk 1 above)
Comment (Sabrucita) [reply src/redux/companies/actions.js]: You're right!

File: src/redux/companies/thunks.js (hunk 2 above)
Comment (Sabrucita) [reply src/redux/companies/thunks.js]: Okay, I fixed it!

**Review by EugenioPonzone — APPROVED** [review]

**Review by BaltazarCortina — APPROVED** [review]

**Pull request merged**